Makefile

```diff
@@ -21,12 +21,11 @@ pull:
 build:
 	govuk-docker build
 
-setup:
-	for repo in $(REPOS); do \
-		make -f $$repo/Makefile; \
-	done
+setup: $(addsuffix _setup,$(REPOS))
 	govuk-docker run whitehall-e2e rake taxonomy:populate_end_to_end_test_data
```

Comment:
This command fails if `REPOS` has been specified and doesn't contain everything - should it be a separate task?